frontend/src/components/eMIB/constants.js

```diff
@@ -44,7 +44,23 @@ export const actionShape = PropTypes.shape({
   reasonsForAction: PropTypes.string,
   task: PropTypes.string,
   emailType: PropTypes.oneOf(Object.keys(EMAIL_TYPE)),
-  emailTo: PropTypes.string,
-  emailCc: PropTypes.string,
+  emailTo: PropTypes.arrayOf(PropTypes.number),
+  emailCc: PropTypes.arrayOf(PropTypes.number),
   emailBody: PropTypes.string
 });
+
+// The stucture of a contact in the addressbook
+export const contactShape = PropTypes.shape({
+  id: PropTypes.number.isRequired,
```

Comment:
If we're just using an array of this shape, is the Id actually required since you have the index? It seems like it might be redundant.